README.md

```diff
@@ -23,7 +23,9 @@ To deploy this project we need to meet the requirements.
 
 ### Requirements
 
-1. Running instance of `Jans Auth Server`, `Jans Fido2`, `Jans Casa` and `Jans Scim`
+1. Running instance of
+   - `Jans Auth Server`-
+   - `Jans Fido2`, `Jans Casa` and `Jans Scim`
```

Comment:
Why is Auth Server on one line, but FIDO, Casa and SCIM are on one line?

Reply:
Solved